FundTrack.BLL/Concrete/OrganizationRegistrationService.cs

```diff
@@ -38,6 +42,15 @@ public OrganizationRegistrationViewModel RegisterOrganization (OrganizationRegis
                     if (checkMembership == null)
                     {
                         var organization = new Organization { Name = item.Name, Description = item.Description };
+
+                        if (!String.IsNullOrEmpty(item.Base64Code))
+                        {
+                            var task = _imgManageService.UploadImage(Convert.FromBase64String(item.Base64Code));

```

Comment:
if items do not upload to Azure, this should not be commited. you have added lines of code to the existing business process (registration of organization) that SHOULD upload images during the registation but in fact, they do nothing. If you want to write some piece of code that is not functional instantly,or does not add any business value, but will in future, do not invoke it in the existing busines flow.